Emby.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -251,14 +245,12 @@ public async Task AcceptCameraUpload(string deviceId, Stream stream, LocalFileIn
 
             if (CameraImageUploaded != null)
             {
-                CameraImageUploaded?.Invoke(this, new GenericEventArgs<CameraImageUploadInfo>
-                {
-                    Argument = new CameraImageUploadInfo
+                CameraImageUploaded?.Invoke(this, new GenericEventArgs<CameraImageUploadInfo>(
```

Comment:
why this code is still there? I thought we removed uploads a while back?

Reply:
🤷‍♂